scrunch/datasets.py

```diff
@@ -1038,10 +993,11 @@ def items(self):
         return zip(self.iterkeys(), self.itervalues())
```

Comment:
this would return a tuple of (var_url, var_shoji) which is something we didn't want, we need (alias, Variable()). Also for the comments above we want the `self._vars` to save the `resource.index.items()` and access it just once; you suggested this some time ago when we first wrote this Mixin class